R/inputs.R

```diff
@@ -0,0 +1,163 @@
+button <- function(name) {
+  function(inputId, ...) shiny.react::reactElement(
+    module = "@/shiny.fluent", name = name,
+    props = shiny.react::asProps(inputId = inputId, ...),
+    deps = shinyFluentDependency()
+  )
+}
+
+#' @rdname Button
```

Comment:
how did you generate this file? anything automated or purely manually?

Reply:
Manually. I'd like to decrease the amount of repetitive typing required, but using R's meta-programming capabilities.